### ISSUE TITLE
[Enh] Add possibility to navigate filesystem of image or container

### ISSUE THREAD

**Issue by JusteRaimbault** [dafnifacility/user-feedback#40]
It would be useful both to
 - be able to navigate the filesystem of the docker image corresponding to a model, for example to check source code of a model provided by an other user
 - be able to navigate the filesystem of the container corresponding to an executed model when this step of the workflow has succeeded, for example to see if result files are as expected

**Comment by JusteRaimbault** [comment]
A related useful enhancement would be to be able to run manually `docker exec` commands in a container (would work for executed models only)

**Comment by BethanPerkins** [comment]
Thanks very much for this Juste. I think we have two different requests here:
- The ability to examine the code executed in a model
- The ability to work (navigate, run commands) in the docker container itself once the model has executed
Would you agree?

**Comment by JusteRaimbault** [comment]
Yes indeed, those are two different features (which I guess correspond to two different technical solutions, one on images, the other on containers); I would say that the second one (work in the container) would be more important, as model source code can be accessed otherwise if model publisher has provided repository information in the metadata.
